### ISSUE TITLE
Remove reset CSS

### ISSUE THREAD

**Issue by nt1m** [PalmerAL/scratchpad#50]
https://github.com/PalmerAL/scratchpad/blob/master/css/editor.css#L13

This ruleset is bad, for the following reasons : 
- Causes bugs (like #49, but that's not the only bug I've seen)
- The * selector is expensive (performance-wise)

**Comment by PalmerAL** [comment]
I would agree, but what is the alternative? Some of these (like the font smoothing and the box-sizing) need to apply to everything. Most of the bugs are caused by the background-color rule, but this exists so that elements can inherit the dark theme color by using <code> background-color: inherit </code> (variables would be perfect for this, but they're only in firefox). Maybe a <code> themeable </code> class?

**Comment by nt1m** [comment]
You could use these classes: 
- theme-accent
- theme-body-bg

You could as well use SCSS and use a task runner (grunt) to compile it.
